vmm/src/lib.rs

```diff
@@ -164,7 +164,8 @@ impl std::fmt::Debug for Error {
             // which is the one coming from KVM at the time.
             // TODO: implement debug for the other errors as well.
             Kvm(ref os_err) => write!(f, "Cannot open /dev/kvm. Error: {}", os_err.to_string()),
-            _ => write!(f, "{:?}", self),
+            GeneralFailure => write!(f, "General failure: something happened because reasons."),
```

Comment:
As we discussed this is a bug. If you do not want to fix it in this PR, you should add an issue. I think it is important to fix this ASAP because if firecracker crashes for whatever reason (besides kvm errors), we have no way of figuring out what happened.

Reply:
I'm also not happy with the error message. Getting a useless error message is frustrating to a user, getting a useless and mocking error message is even more so.

Reply:
Well, you say "mocking," I say "light-hearted."
At a second glance, though, that generic, catch-all error was only thrown in one place, so it was easy enough to get rid of it.

Also fixed the Debug bug. All errors should now be formattable via `{:?}`.